download-data.sh

```diff
@@ -74,30 +172,34 @@ download_data() {
 download_opus_data() {
   LANG_ROOT=$1
   TGT=$2
-
   if [ "$TGT" = "si" ]; then
     URLS=("${SI_OPUS_URLS[@]}")
     DATASETS=("${SI_OPUS_DATASETS[@]}")
-  else
+  elif [ "$TGT" = "ne" ]; then
     URLS=("${NE_OPUS_URLS[@]}")
     DATASETS=("${NE_OPUS_DATASETS[@]}")
+  elif [ "$TGT" = "km" ]; then
+    URLS=("${KM_OPUS_URLS[@]}")
+    DATASETS=("${KM_OPUS_DATASETS[@]}")
+  elif [ "$TGT" = "ps" ]; then
+    URLS=("${PS_OPUS_URLS[@]}")
+    DATASETS=("${PS_OPUS_DATASETS[@]}")
+  elif [ "$TGT" = "fa" ]; then
```

Comment:
We don't have Farsi in the original flores. Is there a reason why are you including it in the target?

Reply:
I believe the intent was to improve the Pashto performance of a multilingual model by training for Farsi as well, being a related language. The Pashto-English parallel corpus was very limited so being able to make use of the Farsi-English corpus would hopefully be a boon. I don't recall whether we tested this or if we did whether it had an impact.